splunk/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/splunk/splunk@sha256:7eca42b8a7a777053e1d0420287a683167ed228ab09444b35b3f4a8f54cb6cd4 AS builder
+FROM docker.io/splunk/splunk@sha256:20b1376a1677ef6eff611b1d1e25e3a170a715c646c2c27711bbb65b03f78ede AS builder
```

Comment:
Is this change needed, or can we save it for another PR?

Reply:
we can save it for another PR, Eran told me that this image should be upgraded.